packages/pds/src/app-view/api/app/bsky/feed/getAuthorFeed.ts

```diff
@@ -64,8 +63,15 @@ export default function (server: Server, ctx: AppContext) {
               .whereRef('post_embed_image.postUri', '=', 'feed_item.postUri'),
           )
       } else if (params.filter === 'posts_no_replies') {
-        // only posts, no replies
-        feedItemsQb = feedItemsQb.where('post.replyParent', 'is', null)
+        feedItemsQb = feedItemsQb
+          // only posts, no replies
+          .where('post.replyParent', 'is', null)
+          // or any reposted replies
+          .orWhere((qb) => {
+            return qb
+              .where('originatorDid', '=', actorDid)
+              .where('type', '=', 'repost')
+          })
```

Comment:
These `where` clauses are cumulative, so could make this kinda brittle if someone needs to edit in the future. Is there a better way to structure this to be more clear to future generations?

Reply:
ugh yeah I think these have to be grouped together under a single `where`

(a new kysely version has a better api around wheres btw, could be worth upgrading soon)

but it'll be simpler to express if we remove the line i mentioned above:
```ts
.where((qb) => 
  qb.where('post.replyParent', 'is', null)
  .orWhere('type', '=', 'repost')
)
```

Reply:
Oh ok yeah I see the distinction there, good call